src/browserlib/extract-elements.mjs

```diff
@@ -137,12 +137,12 @@ export default function () {
   // Extract SVG elements that use the "element-summary" pattern
   const svgSummaryElements = [...document.querySelectorAll('div.element-summary')]
     .map(el => {
-      const dfn = el.querySelector('.element-name dfn');
+      let dfn = el.querySelector('.element-name');
```

Comment:
why s/const/let/ ?
it's a bit confusing to keep the variable name `dfn` if this is no longer targeting the `dfn` element?

Reply:
Ah, good point. I started the patch with an alternative, then realized I didn't need it and forgot to switch back to `const`.